examples/dc/plot_analytic_dipole.py

```diff
@@ -0,0 +1,72 @@
+"""
```

Comment:
I experimented with turning this one into a pure script. The rest I have left as `run()` in the `ifmain`.